### ISSUE TITLE
GitHub Sponsors and other funding sources

### ISSUE THREAD

**Issue by stephen-rowan** [Catalyst-Swarm/Catalyst-Beehive#16]
## GitHub Sponsors - Waiting List 

To apply to be on the https://github.com/sponsors  The-Catalyst-School GitHub organistion must have access to either 

### 1) Organization bank account
This organization uses their own bank account

You can use your organization's bank account or a personal bank account to join GitHub Sponsors. Note that you are responsible for all applicable taxes on payouts you receive. If you use a personal bank account, your country may tax your GitHub Sponsors payouts as personal income. [Learn more](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-organization#submitting-your-bank-information)

### 2) This organization is using a fiscal host
Organizations can use a fiscal host to join GitHub Sponsors in lieu of having their own bank account and legal entity. [Learn more](https://github.blog/2020-03-24-getting-started-with-github-sponsors/#orgnobank)

## opencollective
https://opencollective.com/

## patreon
https://www.patreon.com/

**Comment by stephen-rowan** [comment]
Filip or DominikTilman - here is a link to https://github.com/sponsors/Catalyst-Swarm/waitlist - it specifies a list of countries it supports for bank accounts. Can we provide a bank in one of the supported countries ? If so can the bank account holder register on the GitHub sponsor waiting list with their email ?

**Comment by dominiktilman** [comment]
The given link to the waitlist does not work for me

**Comment by whitevo** [comment]
It works for me, I think only admins can write emails there.
However, under that link, I did not find any country list.
I think this is where the process should be started:
https://github.blog/2020-03-24-getting-started-with-github-sponsors/#orgnobank
Until you know you are eligible and what email address you want to use.

**Comment by stephen-rowan** [comment]
Responding to https://github.com/Catalyst-Swarm/Catalyst-Beehive/issues/16#issuecomment-881852860

That link explains how to display a GitHub Sponsors profile on your Organisation page. The overall starting page is here - https://github.com/sponsors - when you click "Get Sponsored" it takes you to "GitHub Sponsors eligible accounts" and  a list of account you are admin of is listed.

I forgot about the admin requirement - I have now sent a Member invite to @dominiktilman so that he can see the sponsor list.

S

**Comment by dominiktilman** [comment]
@stephen-rowan  

- [ ] Meeting tomorrow wednesday 10 UTC how to set up the profile

**Comment by dominiktilman** [comment]
![Bildschirmfoto 2021-07-21 um 12 08 12 (2)](https://user-images.githubusercontent.com/80850851/126476863-4ae3f39e-4e9f-41f1-b4c5-f3d4e866494d.png)
![Bildschirmfoto 2021-07-21 um 12 40 35 (2)](https://user-images.githubusercontent.com/80850851/126476871-5c18952f-4a15-4c9a-93e1-aa13329a41e8.png)
![Bildschirmfoto 2021-07-21 um 12 40 52 (2)](https://user-images.githubusercontent.com/80850851/126476879-22b8401d-fc59-456b-b445-e3243261361d.png)

**Comment by stephen-rowan** [comment]
Thanks @dominiktilman

**Comment by stephen-rowan** [comment]
> 
> 
> We need to create a strategy/plan on how to get sponsors.

I agree. A lot will depend on promotion of our main open source repository [Catalyst-Beehive](https://github.com/Catalyst-Swarm/Catalyst-Beehive). 

- [ ] Before the repo is promoted its README file will need some refinement - so it points to everything we do (Twitter, YouTube, GitBook, Website etc.)
- [ ] All the open-source documents will need some meaningful content (e.g, our Contribution policy).
- [ ] We will need to demonstrate our open-source innovation - e.g. that we are using the repo to track Swarm's governance.
- [ ] Once the above is in place we can be confident to market the repo at Town Hall, on social media, Cardano Summit etc.

**Comment by stephen-rowan** [comment]
@all-contributors please add @dominiktilman for Funding/Grant Finders

**Comment by stephen-rowan** [comment]
@all-contributors please add @dominiktilman for Funding

**Comment by stephen-rowan** [comment]
@all-contributors please add @dominiktilman for fundingFinding
